target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -78,6 +78,10 @@ netgear,r6800)
 	ucidef_set_led_netdev "lan3" "LAN3" "white:lan3" "lan3"
 	ucidef_set_led_netdev "lan4" "LAN4" "white:lan4" "lan4"
 	;;
+tplink,archer-a6-v3)
+	ucidef_set_led_netdev "lan" "LAN act" "green:lan" "br-lan" 
+	ucidef_set_led_netdev "wan" "WAN link act" "green:wan" "wan" 
```

Comment:
Can you **please** finally remove the "act" here ...

Reply:
done!.